registry/config.go

```diff
@@ -32,6 +32,7 @@ const (
 	REGISTRYSERVER = DEFAULT_V2_REGISTRY
 	INDEXSERVER    = DEFAULT_V1_REGISTRY + "/v1/"
 	INDEXNAME      = "docker.io"
+	NOTARYSERVER   = "https://notary-stage.docker.io"
```

Comment:
In order to merge, this should be pointing at the real/prod notary server.
You can control what backend this is going to via DNS.
I don't see why we should have just one more thing that needs to change in the code here.